From 12cc86a24f7077300ba34a5a04aceec21db08d7b Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 11 Dec 2024 15:47:49 +0800
Subject: [PATCH 262/301] AOSCOS: MIPS: serial: drop STD_FLAGS

Per commit b1d984cf7d6b ("crisv10: use flags from tty_port"), the flags
defined in STD_FLAGS (ASYNC_BOOT_AUTOCONF | ASYNC_SKIP_TEST) were not
tested in other code and therefore needlessly initialised.

Drop them to prevent build errors.

Fixes: "AOSCOS: Revert "staging: sb105x: delete the driver""
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/include/asm/serial.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/mips/include/asm/serial.h b/arch/mips/include/asm/serial.h
index a780ee51a718..2777148dbfc5 100644
--- a/arch/mips/include/asm/serial.h
+++ b/arch/mips/include/asm/serial.h
@@ -12,7 +12,6 @@
  */
 #define BASE_BAUD 0
 #else
-#define STD_COM_FLAGS (ASYNC_BOOT_AUTOCONF | ASYNC_SKIP_TEST)
 #include <asm-generic/serial.h>
 #endif
 
-- 
2.48.1

